### ISSUE TITLE
empty page on pdfroff output

### ISSUE THREAD

**Issue by jarnosz** [jgm/pandoc#7734]
**Current pdf engine for ms format outputs an empty page if --toc is called.**

Tested with manual.txt (the user's manual).

```
pandoc -t ms --eol=lf --toc -Nso manual.pdf manual.txt
```

The generated PDF adds a blank page at the beginning, then the TOC, with some mistakes, particularly at the end, and then the document itself.

The misbehaviour disappears if adding the `--no-toc-relocation` directive.

```
pandoc -t ms --eol=lf --toc --pdf-engine-opt=--no-toc-relocation -Nso manual.pdf manual.txt
```

My suggestion is to add this option directly on the call to pdfroff.

I'll submit a patch, if requested.

**Pandoc version?**
pandoc 2.16.2, on cygwin x86_64 3.20

**Comment by jgm** [comment]
Yes, I can reproduce this.  I wonder if there's another solution?  Using `--no-toc-relocation` by default means that the TOC comes at the end, and this can't be undone (I think) by using `--pdf-engine-opt`, because there is no `--toc-relocation` option.  So currently we can get both behaviors (in the way you illustrated), but if we made the change suggested, only one would be available.

What were the mistakes you saw in the generated TOC?  Did they go away with `--no-toc-relocation`?

**Comment by jgm** [comment]
Ah, I see, the page numbers are wrong, and this does get fixed with `--no-toc-relocation`.

Well, in the absence of any better ideas, I'll accept a PR for this change.

**Comment by jarnosz** [comment]
I believe trying to relocate the toc in the `pdfroff` script was a bad idea to begin with, so in that attempt they produced this bug; but that's a fight for another list.

Meanwhile, adding this directive in pandoc's routine may be an acceptable workaround for the time being.

I'm submitting a patch.
